crc/api/workflow.py

```diff
@@ -255,7 +255,7 @@ def set_current_task(workflow_id, task_id):
     processor = WorkflowProcessor(workflow_model)
     task_id = uuid.UUID(task_id)
     spiff_task = processor.bpmn_workflow.get_task(task_id)
-    cancel_notify = (spiff_task.state == TaskState.COMPLETED and
+    cancel_notify = (hasattr(spiff_task, 'state') and spiff_task.state == TaskState.COMPLETED and
```

Comment:
a spiff_task should always have a state.  Wondering why you needed to add this.

Reply:
I had a weird error when working on the ticket, so I was just being safe.
I can take this out.